engine/xml-config/src/main/resources/job.xsd

```diff
@@ -5,19 +5,26 @@
 
 	<element name="job">
 		<complexType>
-			<sequence>
-				<element name="job-metadata" type="ab:jobMetadataType"
-					minOccurs="0" maxOccurs="1" />
-				<element name="source" type="ab:sourceType" minOccurs="0"
-					maxOccurs="1" />
-				<element name="transformation" type="ab:transformationType"
-					minOccurs="0" maxOccurs="1" />
-				<element name="analysis" type="ab:analysisType" minOccurs="1"
-					maxOccurs="1" />
-			</sequence>
+			<complexContent>
+				<extension base="ab:jobType">
+				</extension>
+			</complexContent>
```

Comment:
I tried that initially actually. But then the JAXB would use as the root element "jobType" instead of a dedicated root "Job". Problem with that is then that when you try to marshall or unmarshall with JAXB it throws an exception because the JobType is not a sure root type. Bah.